consensus/src/chained_bft/block_storage/block_tree.rs

```diff
@@ -28,6 +29,17 @@ use std::{
 };
 use types::crypto_proxies::LedgerInfoWithSignatures;
 
+struct BlockHash {
```

Comment:
minor: `BlockHash` might be an overloaded term and this specific one is misleading (it's not really a block hash). We could call it `BlockPendingVotes` and add comments to the struct.